### PR TITLE
Team 2 shylan21 create profile step 2

### PR DESCRIPTION
**Pull request by Shylan21**

### CODE REVIEW

File: src/context/auth.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ const AuthProvider = ({ children }) => {
 
   const handleRegister = async (email, password) => {
     const res = await register(email, password)
+   console.log('register', res)
```
Comment (procrastian) [inline src/context/auth.js]: remove this console.log

**Review by ManNavic — DISMISSED** [review]
LGTM

**Review by JayVith — DISMISSED** [review]
All changes looking good now, Nice job!

**Review by Auenc — APPROVED** [review]

**Review by procrastian — APPROVED** [review]
LGTM!

**Pull request merged**